go/vt/vttablet/endtoend/main_test.go

```diff
@@ -54,11 +54,12 @@ func TestMain(m *testing.M) {
 			return 1
 		}
 
-		connAppDebugParams, err = hdl.MySQLAppDebugConnParams()
+		connAppDebugParams, err = hdl.MySQLConnParams()
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "could not fetch mysql params: %v\n", err)
 			return 1
 		}
+		connAppDebugParams.Uname = "vt_appdebug"
```

Comment:
Oh. I meant for you to keep that function, but make it use `MySQLConnParams` instead.